packages/perseus/src/perseus-types.ts

```diff
@@ -1,14 +1,40 @@
-// TODO(FEI-4010): Remove `Perseus` prefix for all types here
-// TODO(FEI-4011): Use types generated by https://github.com/jaredly/generate-perseus-flowtypes
+/**
+ * The Perseus "data schema" file.
+ *
+ * This file, and the types in it, represents the "data schema" that Perseus
+ * uses. The @khanacademy/perseus-editor package edits and produces objects
+ * that conform to the types in this file. Similarly, the top-level renderers
+ * in @khanacademy/perseus, consume objects that conform to these types.
```

Comment:
My only feedback is that this seems to confident. It _does_ represent all permutations of the data or it _should_?

But if you feel confident then just 🚢

Reply:
It's aspirational. :)  The new parsing code validates against these types and so I think it's a fair statement.